src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -109,7 +120,13 @@ object ScalafixPlugin extends AutoPlugin {
 
   lazy val cli: Either[Throwable, ScalafixInterface] = {
     try {
-      Right(ScalafixInterface.classloadInstance())
+      val dep = new Dependency(
+        "ch.epfl.scala",
+        s"scalafix-cli_${BuildInfo.scala212}",
```

Comment:
What if we keep the `"g" % "a" %% "v"`syntax and assert that the scalaVersion == BuildInfo.scala212 ? To me this feels more natural, because I always use `%%` for scala library.

Reply:
This is not the syntax for `scalafixDependencies`, this is the coursier-small API that users won't observe

Reply:
See `basic/build.sbt` for an example `scalafixDependencies` usage